### PR TITLE
Update project structure and update makefile

### PR DESCRIPTION
**Pull request by 0x4248**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+CC = g++
+
+CFLAGS = -Wall 
+
+INCLDUE = -I include
```
Comment (0x4248) [inline Makefile]: Include is spelt wrong

Comment (asleepynerd) [reply Makefile]: Fix the include pls

Comment (0x4248) [reply Makefile]: Done

**Comment by asleepynerd** [comment]
Did you put all the code outside of sha1024?

**Comment by 0x4248** [comment]
> Did you put all the code outside of sha1024?

Put it like this
image

**Comment by 0x4248** [comment]
Should be fine to merge now

**Comment by asleepynerd** [comment]
please put it all under sha1024 folder

**Pull request merged**